source/api_c/include/c_api.h

```diff
@@ -94,6 +94,20 @@ extern void DP_DeleteNlist(DP_Nlist* nl);
  */
 const char* DP_NlistCheckOK(DP_Nlist* dp);
 
+/**
+ * @brief The deep potential base model.
```

Comment:
I suggest adding doc `@since API version 24` to all new functions you added in this PR.

https://github.com/deepmodeling/deepmd-kit/blob/edb1e9fecd448512c87806da97b3732c44a0cc2b/source/api_c/include/c_api.h#L13